Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -48,11 +48,25 @@ namespace RDKit {
 typedef std::vector<MatchVectType> VectMatchVectType;
 typedef std::vector<VectMatchVectType> VectVectMatchVectType;
 
+namespace {
```

Comment:
how's the distinction made between what's inside common_properties and what's local SOME_UPPER_CASE_CONSTANT?